File: TD/TD2/Code/3.cpp
/**
 * Afficher la valeur (X / 2), si X est pair; sinon afficher (3 * X + 1)
 */
#include <iostream>

int main()
{
  int a;

  for (int i = 0; i < 20; ++i) {
    std::cout << "Donne moi une valeur : " << std::flush;
    std::cin >> a;

    if (a % 2 == 0)
      a /= 2;
    else
      a = a * 3 + 1;

    std::cout << a << std::endl;
  }

  return 0;
}

File: TD/TD2/Code/Python/3.py
#!/usr/bin/python
# Afficher la valeur (X / 2), si X est pair; sinon afficher (3 * X + 1)
print('Donne moi une valeur : ', end='')
a = int(input())

if a % 2 == 0:
    print(a, '/ 2 =', int(a / 2))
else:
    print('3 *', a, '+ 1 =', int(3 * a + 1))
